### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by niels9001**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,13 +34,11 @@ Quick start guide to building WinUI 3 Gallery locally:
 
 ### 1. Set up the Environment
 
->**Note**: The WinUI 3 Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
-If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here.](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment)
+>**⚠️ Note**: The AI Dev Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
```
Comment (marcelwgn) [inline README.md:37]: I don't think we are the AI Gallery ;)

Comment (niels9001) [reply README.md]: Lol, copy-and-pasting too much! Fixed!

**Review by marcelwgn — APPROVED** [review]

**Comment by Jay-o-Way** [comment]
Let's remove those `en-us/` bits, while we're at it?

File: README.md [hunk 2]
```diff
@@ -34,13 +34,11 @@ Quick start guide to building WinUI 3 Gallery locally:
 
 ### 1. Set up the Environment
 
->**Note**: The WinUI 3 Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
-If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here.](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment)
+>**⚠️ Note**: The WinUI Gallery requires [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later to build and Windows 10 or later to execute.
+If you're building an app with WinUI and Windows App SDK for the first time, follow the [installation instructions here](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here)).
```
Comment (Jay-o-Way) [inline README.md:38]:
```suggestion
If you're building an app with WinUI and Windows App SDK for the first time, follow these [installation instructions](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here).
```

**Pull request closed without merging**